server/src/pages/shot/editor.js

```diff
@@ -210,8 +232,13 @@ exports.Editor = class Editor extends React.Component {
     </div>;
   }
 
-  setColor(color) {
+  setColor(color, colorName) {
     this.setState({color});
+    this.setState({colorName});
+  }
+
+  setTextSize(textSize) {
+    this.setState({textSize});
```

Comment:
Is the text size also in the editor because we want it to remember the user's last selection?  It'd be simpler to have only the text size in the text tool state.

Reply:
That was the thought but I think it should be ok to have it reset to large for every new text annotation